src/main/java/opendcs/dao/ComputationDAO.java

```diff
@@ -171,17 +176,17 @@ private void fillCache()
 
 		try
```

Comment:
fillCache() is called from listCompsForGUI() only if the compCache is empty.  Which seems OK if this is for a UI.  However, if there are other paths to fillCache() clearing the cache may be necessary., perhaps a first step in fillCache() should be to clear the cache?

Reply:
hmm, it's private so I don't think it will but I'll check later on the execution paths. Though even with just for the GUI it should clear the cache, entirely plausible things have been deleted in the meantime.

Reply:
I've put it in for good measure, but it doesn't actually appear needed. The compedit GUI uses the function below this one. And the other usages just call this and it checks if it's ever been called. We'll definitely need to look into this cache later.